src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -259,6 +259,20 @@ class CommandRequestImpl : public CommandImpl,
   mobile_apis::Result::eType PrepareResultCodeForResponse(
       const ResponseInfo& first, const ResponseInfo& second);
 
+  /**
+   * @brief Resolves if the return code must be
+   * UNSUPPORTED_RESOURCE
+   * @param first contains result_code from HMI response and
+   * interface that returns response
+   * @param second contains result_code from HMI response and
+   * interface that returns response.
+   * @return if the communication return code must be
+   * UNSUPPORTED_RESOURCE, function returns true, else
+   * false
+   */
+  bool IsResultCodeUnsupported(const ResponseInfo& first,
```

Comment:
This function is common for all commands and the info parameters may be from all available HMI interfaces (not every time for UI and TTS). That is the reason why I didn't change the names of parameters. I have just added it to the header file.